pegasus/sites.v3/code.org/public/educate/donor-teacher-banner.md.erb

```diff
@@ -0,0 +1,8 @@
+---
+title: Teacher Donor Banner
+theme: responsive
+---
+
+# Teacher Donor Banner
+
+<%= view :teacher_donor_banner, source_page_id: "donor-teacher-banner" %>
```

Comment:
We've got some additional edge cases with how we'll want this to behave on the pegasus page.  Since anyone might load the page (not signed in, signed in as student, signed in as non-targeted teacher) we'll probably want the banner to hide or prompt for sign-in as appropriate in those cases. (This might make sense in a follow-up PR)